critiquebrainz/ws/review/views.py

```diff
@@ -231,21 +233,30 @@ def review_modify_handler(review_id, user):
 
     :resheader Content-Type: *application/json*
     """
-
+    # TODO(psolanki): One caveat is that client will have to pass the unmodified parameter with previous revision's value
+    #                 otherwise it will be set to None
```

Comment:
Need some help to with this. I tried some solutions but they were not promising.

Reply:
Which parameter? I don't quite understand the issue.

Reply:
Suppose the review is of text+rating type. If the user wants to change just the rating and passes just the rating parameter, text will be set to None(because its optional). In order to do so, text (unmodified parameter) must be passed with previous revision's text value. 

Now this does not seem to be a very elegant solution.

Reply:
Since that's an interface for updating reviews I don't see anything wrong without skipping updates for missing fields. If *text* is not provided then it doesn't need to be modified.

Reply:
But, what if the user wants to remove the text part from text+rating review.

Reply:
They can pass `None`.

Reply:
There's a problem with this. Since the parameters are optional, not passing them is equal to passing them with None.

Reply:
Is there a way to check if request JSON has a field or not?

Reply:
Added this check in the parser.